common/DevHomeAdaptiveCards/CardModels/DevHomeSettingsCard.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using AdaptiveCards.ObjectModel.WinUI3;
+using DevHome.Common.DevHomeAdaptiveCards.CardInterfaces;
+using Microsoft.UI.Xaml.Controls;
+using Windows.Data.Json;
+
+namespace DevHome.Common.DevHomeAdaptiveCards.CardModels;
+
+/// <summary>
+/// Represents a settings card that can be rendered through an adaptive card based on the ElementTypeString.
+/// </summary>
+public class DevHomeSettingsCard : IDevHomeSettingsCard
+{
+    // Specific properties for DevHomeSettingsCard
+    // These properties relate to the Windows Community Toolkit's SettingsCard control.
+    // We'll allow extensions to provide the data for the SettingsCard control from an Adaptive Card.
+    // Then we'll render the actual SettingsCard control in the DevHome app.
+    /// <inheritdoc cref="IDevHomeSettingsCard.Description"/>
+    public string Description { get; set; } = string.Empty;
+
+    /// <inheritdoc cref="IDevHomeSettingsCard.Header"/>
+    public string Header { get; set; } = string.Empty;
+
+    /// <inheritdoc cref="IDevHomeSettingsCard.HeaderIcon"/>
+    public string HeaderIcon { get; set; } = string.Empty;
```

Comment:
Does this come through as a base64 string like other ac images?

Reply:
Yea the expectation for the HeaderIcon would be a base64 string